packages/utils/test/abi_encoder/return_values_test.ts

```diff
@@ -173,32 +174,29 @@ describe('ABI Encoder: Return Value Encoding/Decoding', () => {
             // This is by design, as only a struct's contents are encoded and returned by a funciton call.
             expect(decodedReturnValue).to.be.deep.equal(returnValue.fillResults);
         });
-        it('Should decode NULL as default value (single; static)', async () => {
+        it('Should fail to decode NULL (single; static)', async () => {
             // Generate Return Value
             const method = new AbiEncoder.Method(ReturnValueAbis.singleStaticReturnValue);
-            const returnValue = '0x00000000';
             const encodedReturnValue = '0x';
-            const decodedReturnValue = method.strictDecodeReturnValue<string>(encodedReturnValue);
+            const decodeReturnValue = () => method.strictDecodeReturnValue<string>(encodedReturnValue);
             // Validate decoded return value
-            expect(decodedReturnValue).to.be.deep.equal(returnValue);
```

Comment:
For my own edification if nothing else, there's no reason to use `deep.equal` here, since `returnValue` is a string...right?

(Same comment applies to line 201.)

Reply:
Yurrp. I think it was just done for uniformity.

Reply:
Yeah, there was a lot of copy pasta in these tests.